### ISSUE TITLE
New version - Major issues

### ISSUE THREAD

**Issue by raymay813** [opto/nostalgy-xpi#161]
Seems the new version has some major issues. It files the emails in the wrong folders, copy function doesn't work, go to function also broken, instead taking you to a different folder. Is there a way to go back to a previous version?

**Comment by opto** [comment]
it is working fine on win 11.
What OS are you on? Please give better description of the problems if they are not yet in other issues.

I know of three issues: moves only moves the first 100 messages, and the mouse gesture moves a second message if the original is at the top.

And go fails if not in the all view - but that is a TB issue, we use selectfolders with 'forceselect', which, according to TB code, goes there and switches to all view, but it does not always.

Alain,the original author of nostalgy, had some workaround code which does no loger work, because the internal functions which he used have been removed from TB.

**Comment by opto** [comment]
for move > 100 messages and mouse up bounce for mouse gesturees, please try: https://github.com/opto/nostalgy-xpi/releases/tag/v3.2.18

**Comment by opto** [comment]
or this: https://github.com/opto/nostalgy-xpi/releases/tag/v3.2.22

**Comment by opto** [comment]
or this https://github.com/opto/nostalgy-xpi/releases/tag/v3.2.24

**Comment by burval** [comment]
I am seeing the same behavior reported by @raymay813 running version 3.2.17 in TB 91.6.1 (64-bit).  Based on my testing there may be a conflict between your extension and the "QuickFolders" extension.  It would be interesting to know if @raymay813 is also using the "QuickFolders" extension.

**Comment by raymay813** [comment]
@burval , Yes I was using quickfolder but uninstalled. I ended up downgrading TB & Nostology to an older version, works like a charm.

**Comment by opto** [comment]
TB team (John) helped to optimise startup, I am interested whether this solves the problems.
https://github.com/opto/nostalgy-xpi/releases/tag/v3.3.17

Klaus

**Comment by opto** [comment]
new version: https://github.com/opto/nostalgy-xpi/releases/tag/v3.3.19
